projects/spectator/test/form-input/form-input.component.spec.ts

```diff
@@ -4,50 +4,51 @@ import { createHostFactory, SpectatorHost, Spectator, createComponentFactory } f
 import { FormInputComponent } from './form-input.component';
 
 describe('FormInputComponent', () => {
-  let host: SpectatorHost<FormInputComponent>;
```

Comment:
I'm not sure what these changes are.

Reply:
1. On my opinion you have strange example. You have `enableSubnet` input as static `true`. After that you change `enableSubnet` through `setInput`. There you change input of internal component `<app-form-input>` and not host component. Thats why I changed this test. I used `createHost` and `hostProps`.
 2. You had 2 describes with same name. I refactored the code to one top-level describe and 2 more different describes inside it.

Reply:
If you see how to see is better, could you please share your thoughts?